scripts/deploy-with-s3.js

```diff
@@ -22,10 +22,15 @@ const path = require('path')
 const { execSync } = require('child_process')
 const { remove, move, ensureDir } = require('fs-extra')
 const { s3Prefix, s3Bucket, s3Client } = require('./s3-utils')
+const { cleanUpPageData } = require('./page-data-utils')
 
 const rootDir = path.join(__dirname, '..')
```

Comment:
I find `process.cwd()` is more solid than `__dirname` in getting a Gatsby project's root directory, but both methods do work.